lib/workspace.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "user"
+require_relative "channel"
+require "pry"
+
+module SlackCLI
+  class SlackApiError < Exception; end
+
+  class Workspace
+    attr_reader :users, :channels, :selected, :selected, :query
+
+    Dotenv.load
```

Comment:
I noticed you have this line in several places, just maybe there is a way to only need to do this once.  Otherwise well done.